packages/v4/patternfly-docs/content/accessibility/background-image/background-image.md

```diff
@@ -3,11 +3,36 @@ id: Background image
 section: components
 ---
 
-A **background image** allows you to place an image in the background of your page or area of a page.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should not be able to interact with or navigate to the background image, as it is not an interactive element.
+## Accessibility
 
-**Screen reader users** do not need to be able to interact with the background image. However, using alternative text for the screen reader to be able to describe the background image when the user navigates to a page is beneficial.
+To implement an accessible PatternFly **background image**:
 
-## Accessibility application:
 - Check that there is sufficient color contrast between different parts of the background image and any text that will be on the image. Automated testing tools don't often pick up background image colors, so they may not be able to tell you if contrast is sufficient for users with visual disabilities.
+-  Using alternative text so that a screen reader is able to describe the background image when the user navigates to a page is beneficial.
+
+## Testing
+
+ At a minimum, a background image should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="alertGroup-a11y-checkbox-1" label="Users should not be able to interact with or navigate to the background image with the keyboard, as it is not an interactive element." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="alertGroup-a11y-checkbox-2" label="Users do not need to be able to interact with the background image via a screen reader." />
```

Comment:
For the start of the label, "Users should not be able to..." rather than "Users do not need to be able to..." (this might actually depend on what we end up doing regarding the alt text).

Additional nit for the ID's for both items: `backgroundImage-a11y-checkbox-X`